src/main.go

```diff
@@ -16,7 +16,7 @@ func main() {
 	app := gin.Default()
 
 	// Config Init
-	conf := config.LoadConfig("C:\\Progs\\kinza\\src\\config\\config.json")
+	conf := config.LoadConfig("./config/config.json")
```

Comment:
Yes, I fixed the path for Linux. This probably will break your deployment, but dockerfile will be next.